.github/workflows/ci.yaml

```diff
@@ -125,5 +125,5 @@ jobs:
       - name: Build and publish
         uses: lsst-sqre/build-and-publish-to-pypi@v3
         with:
-          python-version: "3.13"
-          pypi-token: ${{ secrets.PYPI_SQRE_ADMIN }}
+          username: ${{ secrets.USERNAME}}
+          password: ${{ secrets.PASSWORD}}
```

Comment:
You'll revert this back to `pypi-token` and set to the name of the PYPI token secret. See the example in the README